<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>380</width>
          <height>61</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>连接操作</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="serialNameBox">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="baudRateBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_connect">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>连接</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_disconnect">
             <property name="minimumSize">
              <size>
               <width>66</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>断开连接</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_refresh">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>门禁监控</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
